Cargo.toml

```diff
@@ -49,6 +44,15 @@ handlebars-iron = "0.22"
 params = "0.8"
 staticfile = { version = "0.4", features = [ "cache" ] }
 
+# Communicating with S3
+rusoto_s3 = "0.43"
+rusoto_core = "0.43"
+rusoto_credential = "0.43"
+
+# Async
+futures = "0.3"
```

Comment:
I would recommend using `default-features = false` with `futures`, or even avoiding the facade and adding a dependency on `futures-util` directly (with `default-features = false` still) to reduce the dependencies it introduces.